students/ryba_paula/lesson_02_flow_control/the_second_maximum.py

```diff
@@ -0,0 +1,15 @@
+largest, second_largest = int(input()), int(input())
```

Comment:
To akurat zbędny krok, ponieważ to program ma znaleźć te liczby. Mamy zapewnione, że użytkownik wprowadzi conajmniej dwie dodatnie liczby i program ma zwrócić druga największą.